templates/ts/non-sso-tab/nodemon.json

```diff
@@ -0,0 +1,9 @@
+{
+  "watch": ["src"],
+  "ext": "*",
+  "exec": "node --import ./ts-node.register.mjs --inspect=9239 src/app.ts",
+  "signal": "SIGINT",
+  "events": {
+    "restart": "npm run build"
```

Comment:
I think you don't need to rebuild the whole project, but just rebuild teamsapp.ts?

Reply:
OK, I'll separate the frontend and backend build processes.